commands/user_test.go

```diff
@@ -190,3 +190,189 @@ func (s *MmctlUnitTestSuite) TestUserCreateCmd() {
 		s.Require().Equal("Unable to update user roles. Error: : Remote error, ", error.Error())
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestUpdateUserEmailCmd() {
+	s.Run("Two arguments are not provided", func() {
+		command := cobra.Command{}
+
+		error := updateUserEmailCmdF(s.client, &command, []string{})
+
+		s.Require().Equal("Expected two arguments. See help text for details.", error.Error())
```

Comment:
I would add as well `s.Len` checks for the amount of lines on `printer.GetLines()` and `printer.GetErrorLines()` slices, to ensure that the printed output is the expected.

When checking the message of an error, you can use [`require.EqualError`](https://godoc.org/github.com/stretchr/testify/require#EqualError):

```suggestion
		s.Require().EqualError(error, "Expected two arguments. See help text for details.")
```

Reply:
I have changed the EqualError method, but I'm not sure if it's worth it adding the printer checks. There is no printing on error cases, so we should be doing something like:

`s.Require().Len(printer.GetLines(), 0)`